bin/determine-tooling

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+"""Determine lint/formatter tool to use."""
+from __future__ import annotations
+
+import argparse
+import os.path
+import sys
+import tomllib
+from typing import Any
+
+parser = argparse.ArgumentParser(
+    description="Find which tool should be used for the given tool type"
+)
+parser.add_argument(
+    "tool",
+    help="Type of tool to decide upon.",
+    choices=["lint", "format", "isort"],
+)
+args = parser.parse_args()
+
+toml_file = "pyproject.toml"
+
+# format and isort require configuration check.
+# flake8 is always the default for "lint".
+if os.path.exists(toml_file):
+    with open(toml_file, "rb") as fd:
+        toml = tomllib.load(fd)
```

Comment:
What happens when using an invalid `toml` file? The workflow ends with a bachtrace?

Reply:
I'm inclined to think that's a feature not a bug -- if the pyproject.toml file is corrupt we really should fix it instead of trying to hide the problem.